src/backroom/pspm_scr2ledalab.m

```diff
@@ -101,23 +101,15 @@
 
 % check output file & save data
 %--------------------------------------------------------------------------
-if exist(outfile, 'file') == 2 && options.overwrite ~= 1
-    if feature('ShowFigureWindows')
-        msg = ['Imported file already exists. Overwrite?', newline, 'Existing file: ',outfile];
-        overwrite = questdlg(msg, 'File already exists', 'Yes', 'No', 'No'); % default not to overwrite by users 
-    else
-        overwrite = 'No'; % default not to overwrite on Jenkins
-    end             
-    close gcf;
-else
-    overwrite = 'Yes';
-end;
-if strcmp(overwrite, 'No')
-    warning('Data discarded ...');
-    return;
+
+if pspm_overwrite(outfile, options)

```

Comment:
[minor comment]
For consistency, shouldn't the syntax be:
```
write_ok = pspm_overwrite(options.output_file, options);
if write_ok
```
as in src/backroom/pspm_find_data_epochs.m line 103?